public/js/controllers/AddressController.js

```diff
@@ -9,6 +9,7 @@ angular.module('BlocksApp').controller('AddressController', function($stateParam
 
     $rootScope.$state.current.data["pageSubTitle"] = $stateParams.hash;
     $scope.addrHash = $stateParams.hash;
+    $scope.checksummedAddrHash = web3.utils.toChecksumAddress($stateParams.hash);

```

Comment:
can we calculate this on backend and then just send to frontend like everything else?

Reply:
Done. I moved it to backend.